--- jdk13u/src/hotspot/share/runtime/os.cpp
+++ jdk13u/src/hotspot/share/runtime/os.cpp
@@ -154,7 +154,7 @@
   // No offset when dealing with UTC
   time_t UTC_to_local = 0;
   if (!utc) {
-#if defined(_ALLBSD_SOURCE) || defined(_GNU_SOURCE)
+#if (defined(_ALLBSD_SOURCE) || defined(_GNU_SOURCE)) && !defined(__sun__)
     UTC_to_local = -(time_struct.tm_gmtoff);
 #elif defined(_WINDOWS)
     long zone;
