--- jdk/make/src/classes/build/tools/generatecurrencydata/GenerateCurrencyData.java.orig	2025-06-02 15:08:19.382852914 +0000
+++ jdk/make/src/classes/build/tools/generatecurrencydata/GenerateCurrencyData.java	2025-06-02 15:08:33.983873597 +0000
@@ -284,9 +284,6 @@
             checkCurrencyCode(newCurrency);
             String timeString = currencyInfo.substring(4, length - 4);
             long time = format.parse(timeString).getTime();
-            if (Math.abs(time - System.currentTimeMillis()) > ((long) 10) * 365 * 24 * 60 * 60 * 1000) {
-                throw new RuntimeException("time is more than 10 years from present: " + time);
-            }
             specialCaseCutOverTimes[specialCaseCount] = time;
             specialCaseOldCurrencies[specialCaseCount] = oldCurrency;
             specialCaseOldCurrenciesDefaultFractionDigits[specialCaseCount] = getDefaultFractionDigits(oldCurrency);
